src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/court/commonplatform/Defendant.kt

```diff
@@ -4,10 +4,10 @@ import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize
 import jakarta.validation.Valid
+import uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner.deserializers.CROIdentifierDeserializer
```

Comment:
not sure about these being in a `prisoner` package - CROs are not only present on prisoners, same goes for PNCs

Reply:
Moved back 👍